### PR TITLE
Sign Up:FE

### PR DESCRIPTION
**Pull request by GabisPapin**
signUp-screen

### CODE REVIEW

File: src/components/signIn/signin.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { FC } from 'react';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+import { useAppDispatch } from '../../redux/hooks';
+import { email, password } from '../../redux/reducers/signIn';
```
Comment (lyudad) [inline src/components/signIn/signin.tsx:5]: connect absolute path no ../ it should be redux/

File: src/components/signIn/signin.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+const signIn: FC = () => {
+  const dispatch = useAppDispatch();
+
+  const formik = useFormik({
```
Comment (lyudad) [inline src/components/signIn/signin.tsx:9]: we discussed - no formic

Comment (GabisPapin) [reply src/components/signIn/signin.tsx]: I changed and also delete the sign in to this branch and created another branch.

File: src/components/signUp/signup.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        <P>{`${t('SignUp.quickSign')}`}</P>
+        <Button>{`${t('SignUp.buttonGoogle')}`}</Button>
+        <P>{`${t('SignUp.or')}`}</P>
+        <P>{`${t('SignUp.textEmail')}`}</P>
+        <Label>
+          {`${t('SignUp.email')}`}
+          <Input
+            id="email"
+            type="email"
+            {...register("email")}
```
Comment (lyudad) [inline src/components/signUp/signup.tsx:51]: on the call I ask to use Controller
https://react-hook-form.com/get-started#IntegratingwithUIlibraries
its more advanced level, and I want that we practice here

File: src/redux/reducers/signUp.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+interface signUpState {
+  email: string,
+  password: string
+}
+
+const initialState: signUpState = {
+  email: '',
+  password: ''
```
Comment (lyudad) [inline src/redux/reducers/signUp.ts:10]: why we need store password in store? 1st its not secure, 2-d it does not make sence

**Comment by GabisPapin** [comment]
Used react-hook-form Controller, delete password in store. Obs: Din't change anything in page design.

**Comment by GabisPapin** [comment]
Created a service/ httpService, implemented the authApi function on the sign up component, delete reduce sign up, correct the style file. Obs: didn't change the screen design.

File: src/components/signUp/signup.styled.ts [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import image from '../../image/google.jpg';
```
Comment (lyudad) [inline src/components/signUp/signup.styled.ts:1]: absolute

**Comment by GabisPapin** [comment]
Corrected the image path, created constants/styles, resolve all conflicts, resolve dependency problems.

**Pull request merged**